src/content/release-notes.mdx

```diff
@@ -5,6 +5,46 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.31.0
+
+4 April 2025
+
+This version is compatible with Kubernetes versions 1.29 to 1.32 \
+Okteto Chart release 1.31 is designed to work with [Okteto CLI 3.6.x](https://github.com/okteto/okteto/releases/tag/3.6.0)
+
+### Deprecation Notice {#deprecation-notice-1.31}
+
+- ⚠️ Important: **Support for Docker Image Manifest Schema 1 images is removed in this** (1.32) due to upstream dependency changes. If you are using older images, they may fail to pull or deploy.\
+[Learn how to check and update your images →](self-hosted/manage/upgrade.mdx#upgrading-to-okteto-131x)
+
+### New Features {#new-features-1.31}
+
+- Added a new Public API endpoint to list users  <!-- 9175 -->
+- Added support for waking up Development Environments based on annotations in Docker Compose files  <!-- 9187 -->
+- Optimized the wake-up process to allow resources and Development Environments to start in parallel
```

Comment:
This won't be included in the release. It is still on review, so it didn't make it